manifests/Quazar Conundrum.yaml

```
@@ -0,0 +1,11 @@
+name: The Quazar Condundrom
+authors: Anerix(Paradox)CorminQuanadol
+homepage: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/
+license: CC-BY-SA-4.0
+version: V0.1.5
+shortDescription: An Odyssey Spanning 100 years
+description: The Quazar Conundrum is a massive century spanning story involving a galaxy-spanning threat, changing hegemony and new factions rising to power.
+url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/Major_ReleaseV015.zip
+autoupdate:
+  type: tag
+  url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum
```

```suggestion
  url: https://github.com/AurthorParadoxQ/The-Quazar-Conundrum/archive/refs/tags/$version.zip
```
this one's my bad, i made a suggestion before you got it right, so it applying that reverted it back to the wrong state.

---

oh great

---

so what should i do to fix it? make a new PR?

---

oh nver mind, i see you fixed it already